inspirehep/modules/forms/static/js/forms/experiments_typeahead.js

```diff
@@ -32,12 +32,14 @@ define([
       remote: {
         url: '/api/experiments?q=experimentautocomplete:%QUERY*',
         filter: function(response) {
-          return $.map(response.hits.hits, function(el) { return el });
-        }
-      },
+          return $.map(response.hits.hits, function(el) { return el }).sort(function(x, y) {
+          return x.metadata.experiment_names[0].title.localeCompare(y.metadata.experiment_names[0].title);
```

Comment:
This line is logically nested to the previous line, so it should also be nested in the code. Otherwise one sees two consecutive `return` statements and thinks that there's a bug.